### PR TITLE
Agendas make it optional event description

### LINKED ISSUE #1429: Agendas / Make it optional to import event description
Add a checkbox: "Don't import event description"
Also, move the "delete this calendar conf" to the bottom of the page

### PR DESCRIPTION
**Pull request by apradillap**
Closes #1429 

### What does this PR do?

- Add a checkbox: "Don't import event description"
- Also, move the "delete this calendar conf" to the bottom of the page
- I have completed the tests to import with description and without description

### How should this be manually tested?

From Agenda configuration change the attribute and then in events, you will see them without description

captura de pantalla 2018-02-13 a las 17 35 18

### CODE REVIEW

**Comment by apradillap** [comment]
Tests:

- [x] Google Calendar
- [x] Lotus Notes
- [ ] Exchange: we haven't been able to test local or staging even with end-user credentials.

File: app/models/gobierto_calendars/calendar_configuration.rb [hunk 1]
```diff
@@ -10,5 +10,13 @@ class CalendarConfiguration < ApplicationRecord
     belongs_to :collection, class_name: "GobiertoCommon::Collection"
 
     validates :collection_id, :integration_name, :data, presence: true
+
+    def without_description
```
Comment (ferblape) [inline app/models/gobierto_calendars/calendar_configuration.rb:14]: 👍

**Review by ferblape — APPROVED** [review]
It's good to me.

**Comment by ferblape** [comment]
@furilo please review how e have organized the options in the calendar configuration:

screen shot 2018-02-14 at 08 31 45

**Pull request merged**